src/components/currentData.js

```diff
@@ -0,0 +1,85 @@
+import React, { PureComponent, PropTypes } from 'react'
+import { connect } from 'react-redux'
+import * as actions from '../actions/actionIndex'
+import reactCSS from 'reactcss'
+
+const styles = reactCSS({
+  default: {
+    currentData: {
+      float: 'left',
+      width: '49.8%',
+      marginBottom: 0,
+      backgroundColor: '#ef4856',
+      height: 240,
+    },
+    container: {
+      float: 'top',
+    },
+    temperatureText: {
+      fontSize: 50,
+      marginBottom: 10,
+      marginTop: 2,
+    },
+
+  }
+})
+
+const { object, func } = PropTypes
+
+class CurrentData extends PureComponent {
+
+  static PropTypes = {
+    todaysSummary: object,
+    loadTodaysSummary: func.isRequired,
+  }
+
+  static defaultProps = {
+    todaysSummary: {},
+  }
+
+  componentWillMount() {
+    this.props.loadTodaysSummary()
+  }
+
+  render() {
+
+    if(this.props.todaysSummary != null) {
```

Comment:
Because you're accessing `todaysSummary` many times throughout this `render()`, it's best to destructure it. e.g.

```js
render() {
  const { todaysSummary } = this.props

  if (todaysSummary) {
    ...
  }
}